bigchaindb/commands/utils.py

```diff
@@ -74,12 +74,50 @@ def start_logging(args):
     return start_logging
 
 
+def _convert(value, default=None, convert=None):
```

Comment:
It would be nice to use the "keyword-only arguments" approach as outlined in PEP 3102. That is:

```python
def _convert(value, *, default=None, convert=None)
```

This helps making the code clearer.